doozerlib/runtime.py

```diff
@@ -29,8 +29,8 @@
 from multiprocessing import Lock
 from repos import Repos
 import brew
-import constants
 from exceptions import DoozerFatalError
+import kinit
```

Comment:
If the one `kinit` function in that new module is only used here, is there a chance you could move it here **in the future**? Less module namespace sprawl, localization of only-used-once stuff and all that jazz.

Reply:
I removed the whole thing. Didn't really work as desired anyways.